po/de.po

```diff
@@ -479,15 +482,15 @@ msgstr ""
 
 #: ui/createnet.ui:788
 msgid "<b>DNS domain name</b>"
-msgstr ""
+msgstr "<b>DNS Domänenname</b>"
 
 #: ui/createpool.ui:9
 msgid "Add a New Storage Pool"
 msgstr "Neuen Speicher-Pool hinzufügen"
 
 #: ui/createpool.ui:50
 msgid "<span size='large'>Create storage pool</span>"
-msgstr ""
+msgstr "<span size='large'>Ersteke Speicherpool</span>"
```

Comment:
Typo. Should be `Erstelle Speicherpool`

Reply:
Translations are handled on weblate. Please make the change there otherwise it will be lost: https://translate.fedoraproject.org/projects/virt-manager/virt-manager/